internal/bson/bson_test.go

```diff
@@ -149,7 +149,7 @@ func fuzzBinary(f *testing.F, testCases []testCase, newFunc func() bsontype) {
 			br := bytes.NewReader(b)
 			bufr := bufio.NewReader(br)
 			if err := v.ReadFrom(bufr); err != nil {
-				t.Skip(err)
+				t.Skip()
```

Comment:
Skip's argument is ignored by the fuzzing engine and only slows it down a tiny bit